app/server/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
     "compilerOptions": {
-        "target": "es2019",
+        "target": "esnext",
+        "module": "nodenext",
```

Comment:
Specify latest es and module support - this allows both `import`s and `require`s, which seemed to be the main problem in the docker container. I'm not sure why this had stopped working for docker, but ran ok on metal. :shrug: